uplc.md

```diff
@@ -27,7 +44,8 @@ module PLUTUS-CORE-LEXICAL-GRAMMAR
    syntax BuiltinName  ::= Name                                  // builtin term name
 // syntax Integer      ::= r"[+-]?[0-9]+"                [token] // int
    syntax ByteString   ::= r"#([a-fA-F0-9][a-fA-F0-9])+" [token] // hex string
-   syntax Version      ::= r"[0-9]+(.[0-9]+)*"           [token] // version
+// syntax Version      ::= r"[0-9]+(.[0-9]+)*"           [token] // version
```

Comment:
We can just remove this line. If you want to keep track of it, maybe put it in the comment starting on line 18

Reply:
OK